Formula/qt.rb

```diff
@@ -23,134 +23,210 @@ class Qt < Formula
     sha256 cellar: :any, mojave:        "25c4a693c787860b090685ac5cbeea18128d4d6361eed5b1bfed1b16ff6e4494"
   end
 
-  keg_only "Qt 5 has CMake issues when linked"
-
+  depends_on "cmake" => [:build, :test]
+  depends_on "ninja" => :build
   depends_on "pkg-config" => :build
   depends_on xcode: :build
-  depends_on macos: :sierra
 
-  uses_from_macos "gperf" => :build
-  uses_from_macos "bison"
-  uses_from_macos "flex"
+  depends_on "assimp"
+  depends_on "dbus"
+  depends_on "double-conversion"
+  depends_on "freetype"
+  depends_on "glib"
+  depends_on "icu4c"
+  depends_on "jasper"
+  depends_on "jpeg"
+  depends_on "libb2"
+  depends_on "libpng"
+  depends_on "libproxy"
+  depends_on "libtiff"
+  depends_on "llvm"
+  depends_on "mysql"
+  depends_on "pcre2"
+  depends_on "postgresql"
+  depends_on "python@3.9"
+  depends_on "unixodbc"
+  depends_on "webp"
+  depends_on "zstd"
```

Comment:
We should remove any of the ones here that are optional.

Reply:
Optional dependencies I have wrote in the pull request.

Reply:
jasper, jpeg, webp and libtiff stay here because there is plugins in qt@5, which use bundled version, and in qt, they are not distributed in qt-everywhere.